plugins/repository-hdfs/hadoop-client-api/build.gradle

```diff
@@ -2,53 +2,32 @@ import org.gradle.api.file.ArchiveOperations
 
 apply plugin: 'elasticsearch.java'
 
-sourceSets {
-  patcher
-}
+def patched = Attribute.of('patched', Boolean)
```

Comment:
I think this whole project is not needed anymore and we can transform directly in the repository-hdfs project (parent)